examples/timestamping/backend/src/api.rs

```diff
@@ -21,28 +21,37 @@ use schema::{Schema, TimestampEntry};
 use transactions::TxTimestamp;
 use TIMESTAMPING_SERVICE;
 
+/// Describes query parameters for `handle_timestamp` and `handle_timestamp_proof` endpoints.
 #[derive(Debug, Clone, Copy, Serialize, Deserialize)]
 pub struct TimestampQuery {
+    /// Hash of the requested timestamp.
     pub hash: Hash,
 }
 
 impl TimestampQuery {
+    /// Creates new `TimestampQuery` with given `hash`.
     pub fn new(hash: Hash) -> Self {
         TimestampQuery { hash }
     }
 }
 
+/// Describes the information to prove the correctness of the timestamp entry.
```

Comment:
I'm not sure I understand the sentence, is this description required to prove the correctness or this information is required to prove the correctness?
If its the second case, 'Describes the information required to prove' would perhaps be better.